src/api/app/models/project.rb

```diff
@@ -251,6 +258,49 @@ def cleanup_linking_targets
     end
   end
 
+  def update_meta(request_data, params)
+    result = false
+
+    Project.check_target_of_link(name, request_data)
+    Project.update_project_meta_maintains(request_data)
```

Comment:
What are those for?

beside, we are [not catching](https://github.com/openSUSE/open-build-service/pull/1102/files#diff-86772a8e6d058f9f138b75ac35927ffdR674) the exceptions that could be thrown.